packages/plugin-dlx/sources/commands/dlx.ts

```diff
@@ -43,7 +43,16 @@ export default class DlxCommand extends BaseCommand {
 
       await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`package.json`)), `{}\n`);
       await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`yarn.lock`)), ``);
-      await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`.yarnrc.yml`)), `enableGlobalCache: true\n`);
+
+      const localConfigPath = ppath.join(ppath.cwd(), toFilename(`.yarnrc.yml`));
```

Comment:
I think it would be better to use `projectCwd` for this (obtained through `await Configuration.findProjectCwd(this.context.cwd, Filename.lockfile);`), since `ppath.cwd()` will return the path of a workspace if the command is run within a workspace.